src/mbgl/gl/context.cpp

```diff
@@ -124,10 +124,27 @@ void Context::initializeExtensions(const std::function<gl::ProcAddress(const cha
             return nullptr;
         };
 
-        debugging = std::make_unique<extension::Debugging>(fn);
-        if (!disableVAOExtension) {
```

Comment:
We're using `disableVAOExtension` in https://github.com/mapbox/mapbox-gl-native/blob/8ec92b72473305cbf14c31b185fd5634ce670837/test/map/map.test.cpp#L450-L459
so it needs to be re-added to the exclusion logic below